tests/db_functions/json/test_json_object.py

```diff
@@ -0,0 +1,49 @@
+from django.db.models.expressions import F
+from django.db.models.functions import JSONObject, Lower
+from django.test import TestCase
+
+from ..models import Author
+
+
+class JSONObjectTests(TestCase):
```

Comment:
This function is not supported by all databases
```suggestion
@skipUnlessDBFeature('supports_json_field')
class JSONObjectTests(TestCase):
```